test/unittests/parser_test.cpp

```diff
@@ -138,7 +138,7 @@ TEST(parser, code_locals_invalid_type)
     EXPECT_THROW_MESSAGE(parse(wasm), parser_error, "invalid valtype 123");
 }
 
-TEST(parser, empty_module)
+TEST(parser, module_empty)
```

Comment:
Perhaps you want to merge this single-line-change commit into the next commit changing parser tests.

Reply:
Done.